dashboards/overview.json

```diff
@@ -748,7 +748,7 @@
                "steppedLine": false,
                "targets": [
                   {
-                     "expr": "sum(rate(grpc_server_handled_total{grpc_code!=\"OK\",namespace=\"$namespace\",job=~\"thanos-store.*\",grpc_type=\"unary\"}[$interval])) / sum(rate(grpc_server_started_total{namespace=\"$namespace\",job=~\"thanos-store.*\",grpc_type=\"unary\"}[$interval]))",
+                     "expr": "sum(rate(grpc_server_handled_total{grpc_code~=\"Unknown|ResourceExhausted|Internal|Unavailable\",namespace=\"$namespace\",job=~\"thanos-store.*\",grpc_type=\"unary\"}[$interval])) / sum(rate(grpc_server_started_total{namespace=\"$namespace\",job=~\"thanos-store.*\",grpc_type=\"unary\"}[$interval]))",
```

Comment:
Is Unknown really an error? I'd compare it more likely with a 4xx status code on HTTP

Reply:
When it translated to HTTP domain, I guess in maps to 5XX errors. Hence I kept it.

Reply:
Used resources: https://github.com/grpc/grpc/blob/master/doc/statuscodes.md and https://github.com/grpc/grpc/blob/master/doc/http-grpc-status-mapping.md